metricbeat/beater/module.go

```diff
@@ -0,0 +1,204 @@
+package beater
```

Comment:
The eventBuilder is very specific to metricbeat. In what use cases do you envision it being reusable as a library?